iex_trading/IEX_stocks.ipynb

```diff
@@ -198,19 +174,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In the `visualize_symbol_raster` function, the `.select` method on our\n",
-    "`spikes` object is used to select only the spikes that match the symbol\n",
-    "specified in the argument. This function also take an integer `offset`\n",
-    "argument that offsets the rasterized `Image` vertically by one unit (the\n",
-    "spikes are unit length as specified with `spike_length=1`).\n",
-    "\n",
-    "One other difference from the previous notebook is that now a datashader\n",
-    "aggregator over the `'size'` column is used in order to visualize the\n",
-    "trade volume as opposed to the trade count.\n",
-    "\n",
-    "Now we can use this function with our two chosen stock symbols (PINS and\n",
-    "CHK) to create an overlay. Lastly, we want to use the y-axis to label\n",
-    "these stocks so we use a custom `yticks` option and set the `ylabel`.\n"
+    "In the `visualize_symbol_raster` function, the `.select` method on our `spikes` object is used to select only the spikes that match the symbol specified in the argument. This function also takes an integer `offset` argument that offsets the rasterized `Image` vertically by one unit (the spikes are unit length as specified with `spike_length=1`).\n",
+    "\n",
+    "Another difference from the previous notebook is the Datashader aggregator over the `'size'` column is used to visualize the trade volume as opposed to the trade count.\n",
```

Comment:
Just to make it consistent every time you referred to the previous notebook:

```suggestion
    "Another difference from the [previous notebook](./IEX_trading.ipynb) is the Datashader aggregator over the `'size'` column is used to visualize the trade volume as opposed to the trade count.\n",
```

Reply:
Thanks, I'll make that change.